app/src/main/java/fr/neamar/kiss/result/Result.java

```diff
@@ -241,4 +333,10 @@ public int getThemeFillColor(Context context) {
         ta.recycle();
         return color;
     }
+
+    public long getUniqueId()
+    {
+        //TODO: this must be changed! hash codes are not unique! This was used just for fast development of a feature
```

Comment:
I believe we can consider hashCode unique enough in this context, since we're never going to display more than a couple hundreds record at a time and collisions are highly unlikely in this scenario